DateTime.ts

```diff
@@ -15,4 +15,12 @@ export namespace DateTime {
 	export function now(): DateTime {
 		return create(new Date())
 	}
+	export function localize(value: Date, locale?: string, options?: object): DateTime {
```

Comment:
We should create a `Locale` type in isoly, or? @fredrikbryntesson, what do you think?

Reply:
This remains to be done.